browse/templates/archive/archive_list_all.html

```diff
@@ -0,0 +1,43 @@
+{%- extends "base.html" -%}
+
+{% block title %}Archives{% endblock %}
+
+{% block header_h1 %}
+<h1>> archives</h1>
```

Comment:
link to arXiv.org missing from the `header_h1` block:
<img width="477" alt="screenshot 2019-02-26 14 55 31" src="https://user-images.githubusercontent.com/746253/53442184-b2f9a300-39d6-11e9-9fae-0e869e14692c.png">

(also happening on individual archive pages)

Reply:
This use to be handled by the base template. Should I just hardcode this?